scripts/insert_into_appointments.sql

```diff
@@ -1 +1,6 @@
 -- insert appointments creating connections between doctors and patients
+INSERT INTO appointments(patient_id, doctor_id, appt_date, appt_time, comments) VALUES
+  (9,5, '2016-08-12', 10:00),
+  (6,7, '2016-08-17', 13:00),
+  (9,7,'2016-08-12', 10:00),
+  (8,5,'2016-08-12', 13:09);
```

Comment:
This would only work if you knew the ID's of the doctors and patients you wanted to use. Here's another way to do it:

```sql
-- insert appointments creating connections between doctors and patients
INSERT INTO appointments(date, time, doctor_id, patient_id)
  SELECT DATE '2017-08-12', TIME '1000',
    (SELECT id FROM doctors WHERE family_name = 'Hilton'),
    (SELECT id FROM patients WHERE family_name = 'Wilcox')
  UNION
  SELECT '2017-08-17', '1300',
    (SELECT id FROM doctors WHERE family_name = 'Hubbard'),
    (SELECT id FROM patients WHERE family_name = 'Talley')
  UNION
  SELECT '2017-08-17', '1000',
    (SELECT id FROM doctors WHERE family_name = 'Hubbard'),
    (SELECT id FROM patients WHERE family_name = 'Wilcox')
  UNION
  SELECT '2017-08-17', '1300',
   (SELECT id FROM doctors WHERE family_name = 'Hilton'),
   (SELECT id FROM patients WHERE family_name = 'Hodge')
;
```